kubernetes/baseline-attack.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -eu
+# shellcheck disable=SC2001
+
+RUN_NAME=$1
+
+python3 go_attack/scripts/baseline_attack.py \
+  --config go_attack/configs/gtp-base.cfg \
+  --models /shared/victims/kata1-b40c256-s11840935168-d2898845681.bin.gz \
+    /shared/victims/kata1-b20c256x2-s5303129600-d1228401921.bin.gz \
+  --num-games 150 \
+  --policy edge mirror spiral random \
+  --num-visits 1 2 4 8 16 32 \
+  --passing-behavior standard avoid-pass-alive-territory \
+  --victim-color B W \
+  --moves-before-pass 800 \
+  --parallel-runs-per-gpu 12 \
+  --log-dir "$RUN_NAME"/original
```

Comment:
kinda inflexible to have all these params hardcoded, but we don't rerun baseline attacks very often anyway so I think it's not a big deal to have to just edit this file whenever we want to run baseline attacks with different params